test/integration/expression/expression.test.ts

```diff
@@ -52,7 +51,7 @@ describe('expression', () => {
                 delete fixture.metadata;
 
                 const fname = path.join(__dirname, expressionTestFileName);
-                fs.writeFileSync(fname, `${stringify(fixture)}\n`);
+                fs.writeFileSync(fname, JSON.stringify(fixture));
```

Comment:
This is writing everything is a single line, as can be seen in the linked commit.
I suggest to change it to:
```suggestion
                fs.writeFileSync(fname, JSON.stringify(fixture, null, 4));
```

Reply:
I use 2 space for format.